pkg/registry/apis/provisioning/history.go

```diff
@@ -58,6 +60,12 @@ func (h *historySubresource) Connect(ctx context.Context, name string, opts runt
 	}
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		hist, ok := repo.(repository.Historical)
```

Comment:
Should we use this new interface or `VersionedRepository`?

https://github.com/grafana/grafana/blob/87a8d49e3a89813128d0dd8b8e3132b5df1908c0/pkg/registry/apis/provisioning/repository/repository.go?plain=1#L123-L129